openstudiocore/src/model/HeatPumpWaterToWaterEquationFitCooling.hpp

```diff
@@ -60,16 +62,24 @@ class MODEL_API HeatPumpWaterToWaterEquationFitCooling : public WaterToWaterComp
   //@{
 
   /** In EnergyPlus 8.7.0 and above this field maps to the EnergyPlus field named "Reference Load Side Flow Rate" **/
-  double ratedLoadSideFlowRate() const;
+  boost::optional<double> ratedLoadSideFlowRate() const;
```

Comment:
This is a breaking API change and I don't think that is OK. I'm not sure what the right solution is. Clearly we need to support autosizing, and normally that would mean returning optional.

One idea is to return a sentinel like -999 for autosize, but that would be very non standard for OpenStudio. On the other hand it wouldn't break the API. @macumber I'm confident you will have an opinion, so lets hear it please.